cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -152,6 +152,7 @@ purge() {
     rm -rf /var/lib/snapd/mount/*
     rm -rf /var/lib/snapd/sequence/*
     rm -rf /var/lib/snapd/apparmor/*
+    rm -rf /var/lib/snapd/inhibit/*
```

Comment:
Do we need a similar line in `snapd.postrm` in deb packaging?

Reply:
No because it does rm -rf after the `aux` fiasco before IIRC.